netsgiro/enums.py

```diff
@@ -23,15 +24,15 @@ class RecordType(IntEnum):
     TRANSACTION_AMOUNT_2 = 31
     TRANSACTION_AMOUNT_3 = 32  # Only for TransactionType 20 and 21
     TRANSACTION_SPECIFICATION = 49
-    AGREEMENTS = 78         # TODO Better name?
+    TRANSACTION_AGREEMENTS = 70  # TODO Better name?
     ASSIGNMENT_END = 88
     TRANSMISSION_END = 89
 
 
 class AvtaleGiroAssignmentType(IntEnum):
-    PAYMENT_REQUEST = 0     # TODO Better name?
+    PAYMENT_REQUESTS = 0    # TODO Better name?
     AGREEMENTS = 24         # TODO Better name?
-    CANCELATION = 36        # TODO Better name?
+    CANCELATIONS = 36       # TODO Better name?
```

Comment:
I'll fix this in the next PR. Fixing it now causes more conflicts with later commits than I want to resolve.